rule/error-return.go

```diff
@@ -47,6 +47,9 @@ func (w lintErrorReturn) Visit(n ast.Node) ast.Visitor {
 	if len(ret) <= 1 {
 		return w
 	}
+	if isIdent(ret[len(ret)-1].Type, "error") {
+		return true
```

Comment:
This line does not compile.
Must be: `return nil`

Reply:
Fixed.